backend/e4h-services/ingestion-service/app/api/endpoints/template_generation.py

```
@@ -142,7 +144,137 @@ async def get_facility_ingestion_template_with_data(
+            media_type="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
+        )
+
+    except Exception as e:
+        logger.error(f"Unhandled error in get_facility_ingestion_template: {e}")
+        raise HTTPException(status_code=500, detail=f"An unexpected error occurred: {str(e)}")
+
+
+@router.post('/fieldplanFacilityIngestionTemplate',
+            summary='Generate facility ingestion template Excel file with schema, already present data and boundary codes',
+            response_description="Returns Excel template with facility schema, facility data and boundary codes")
+async def get_facility_ingestion_template_with_data(
```

Ruff (0.13.3)

Redefinition of unused `get_facility_ingestion_template_with_data` from line 47

(F811)